shared/chat/conversation/input-area/normal/container.js

```diff
@@ -233,6 +231,5 @@ export default compose(
         this.props.inputFocus()
       }
     },
-  }),
-  mentionHoc
-)(Input)
+  })
+)(MentionHocInput)
```

Comment:
Even with `MentionHocInput` typed, all the above is basically untyped, even the supported `recompose` functions like `withHandler`